database/domains.py

```diff
@@ -37,6 +37,8 @@ class Domains:
         "Ruin Serpent": None,
         "Electro Regisvine": None,
         "Jadeplume Terrorshroom": None,
+        "Aeonblight Drake": None,
+        "Algorithm of Semi-Intransient Matrix of Overseer Network": None,
```

Comment:
Also I'll be adding world bosses to the API soon, so in the future, all data is centralized within the API